help/include/to-do-lists-examples.md

```diff
@@ -0,0 +1,21 @@
+### What you type
+
+```
+/todo
+```
+
+!!! warn ""
+
+    The only content in the message must be `/todo`. Otherwise, the message will
+    be sent as a regular message and the to-do list will not be created.
```

Comment:
This is inaccurate -- a `/todo` message's remaining content will be ignored.

Reply:
(It's probably worth always testing this sort of details whenever you're touching documentation that mentions it -- it's the kind of thing that can turn out to be stale because someone changed the implementation and didn't realize that detail was documented).

Reply:
Good catch! It looks like the implementation did change a bit since the last time I tested it. I'll keep in mind that I should retest if the PR has been around for a while.